### PR TITLE
[CP-1392] Moving device connection logic to app module

### PR DESCRIPTION
**Pull request by igorbezsmertnyi**
Jira: [CP-1392]

**Description**
Moving device connection logic into app module

<details>
<summary><b>Screenshots</b></summary>
// put images here
</details>

[CP-1392]: https://appnroll.atlassian.net/browse/CP-1392?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/app/src/device/constants/device-communication-event.constant.ts [hunk 1]
```diff
@@ -4,6 +4,8 @@
  */
 
 export enum DeviceCommunicationEvent {
-  Disconnected = "disconnected",
-  DataReceived = "dataReceived",
+  Connected = "device-connected",
+  Disconnected = "device-disconnected",
+  DataReceived = "device-data-received",
+  InitializationFailed = "device-connection-failed",
```
Comment (dkarski) [inline packages/app/src/device/constants/device-communication-event.constant.ts:10]: 😉 
```suggestion
  InitializationFailed = "device-initialization-failed",
```
or
```suggestion
  ConnectionFailed = "device-connection-failed",
```

**Review by dkarski — APPROVED** [review]
💪

**Review by dkarski — CHANGES_REQUESTED** [review]
the phone does not connect to the application. Did you test your changes from the regression perspective? 🤔 
Zrzut ekranu 2022-10-26 o 08 16 40


Zrzut ekranu 2022-10-26 o 08 18 38

**Comment by igorbezsmertnyi** [comment]
@dkarski Yes, I tested it.

**Comment by igorbezsmertnyi** [comment]
@dkarski 

https://user-images.githubusercontent.com/21984930/197949992-a5a93c4a-136d-4451-bff9-782324c0e666.mov

**Comment by dkarski** [comment]
> @dkarski Yes, I tested it.

Please test the connection with passcode logic (without a passcode connection works almost properly).

> almost properly
almost because the logs are huge!

**Comment by igorbezsmertnyi** [comment]
@dkarski https://github.com/mudita/mudita-center/pull/1037/commits/520f38af4b984f325fa5889fef332f0132ebc1eb

**Review by wojcik92michal — APPROVED** [review]

**Review by dkarski — APPROVED** [review]

**Pull request merged**